tribute_page.html

```diff
@@ -1,20 +1,22 @@
-<body id='main'>
-  <header>
-    <section id="title">
```

Comment:
It's a good developer skill to create a base file to start your projects. With that the job will be more quickly and easy to start.